app/components/Settings/UserConnections/UserConnections.react.js

```diff
@@ -57,9 +66,22 @@ export default class UserConnections extends Component {
 	render() {
 
 		const {connectionObject, updateConnection} = this.props;
-		const inputs = CONNECTION_CONFIG[connectionObject.dialect]
+        const sampleCredentialsStyle = {
+            lineHeight: 1,
+            textAlign: 'left',
+            width: '100%',
+            maxWidth: 200,
+            display: 'inline-block'
```

Comment:
could be a 1-liner: `display: this.state.showSampleCredentials ? 'inline-block': 'none'`